extrinsic_T: [ 0.0, 0.0, 0.36 ]
extrinsic_R: [  1, 0, 0,
                0, 1, 0,
                0, 0, 1 ]